src/main/java/core/basesyntax/dao/FruitStorageDaoImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class FruitStorageDaoImpl implements FruitStorageDao {

```

Comment:
```suggestion
public class FruitDaoImpl implements FruitDao {
```
Dao layer represents an access to the object. For example, it is the same to say FruitMysqlDao. We don't need this 'Mysql' and 'storage' because its obvious that we use it behind the dao layer